src/test/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2IntegrationTest.java

```diff
@@ -63,6 +66,49 @@ public void testDreamTumorNormal(final File tumorBam, final String tumorSample,
         Assert.assertTrue(fdr < 0.5);
     }
 
+    // make a pon with a tumor and then use this pon to call somatic variants on the same tumor
+    // if the pon is doing its job all calls should be filtered by this pon
+    @Test(dataProvider = "dreamSyntheticDataSample1")
+    public void testPon(final File tumorBam, final String tumorSample, final File normalBam, final String normalSample) throws Exception {
+        Utils.resetRandomGenerator();
+
+        final File ponVcf = createTempFile("pon", ".vcf");
+        final String[] createPonArgs = {
+                "-I", tumorBam.getAbsolutePath(),
+                "-tumor", tumorSample,
+                "-I", normalBam.getAbsolutePath(),
+                "-normal", normalSample,
+                "-R", b37_reference_20_21,
+                "-L", "20",
+                "-O", ponVcf.getAbsolutePath(),
+                "-artifact_detection_mode"
+        };
+
+        runCommandLine(createPonArgs);
+        new Main().instanceMain(makeCommandLineArgs(Arrays.asList("-F", ponVcf.getAbsolutePath()), "IndexFeatureFile"));
```

Comment:
As discussed, please describe that this command creates an index file (or does it make the index file visible?) in a comment